sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/ResourceDetails.java

```diff
@@ -55,7 +55,7 @@ private void setCustomDocumentModelLimit(int customDocumentModelLimit) {
      *
      * @return the customNeuralDocumentModelBuilds value.
      */
-    public QuotaDetails getCustomNeuralDocumentModelBuilds() {
+    public QuotaDetails getCustomNeuralDocumentModelQuotaDetails() {
```

Comment:
This would be super long for us to use in Python, but it is clearer :P

Reply:
- Maybe we could omit the "Custom" part since that's implied from "Neural"?
- "DocumentModel" could be reduced to "Model" only but I think we should keep it if we wanted to be consistent with the rest of the public API.
- Maybe omitting "Details"?

`getNeuralModelQuota` or `getNeuralDocumentModelQuota`?

Reply:
Talked to Anna, we can go with `neural_document_model_quota` in Python